src/main/java/org/opentripplanner/middleware/otp/core/util/IntUtils.java

```diff
@@ -0,0 +1,179 @@
+package org.opentripplanner.middleware.otp.core.util;
+
+import java.util.List;
+import java.util.ArrayList;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class IntUtils {
+    private static final Logger LOG = LoggerFactory.getLogger(IntUtils.class);
+    public  static final String POINT_PREFIX = "POINT("; 
+    
+    /** does this string appear to be a coordinate of some sort */
+    public static boolean looksLikeCoordinate(String str)
+    {
+        if(str != null && (str.contains(POINT_PREFIX) || str.matches("[\\s]*[0-9\\-.]+[,\\s]+[0-9\\-.]+[\\s]*")))
```

Comment:
This might also be from the OTP source, but I would add brackets to multi-line ifs throughout the file.